client/src/pages/private/ParticipantTable.js

```diff
@@ -123,19 +74,33 @@ export default () => {
   const [columns, setColumns] = useState(defaultColumns);
   const [locationFilter, setLocationFilter] = useState(null);
   const [siteSelector, setSiteSelector] = useState(null);
-  const [fsaFilter, setFsaFilter] = useState(null);
-  const [fsaText, setFsaText] = useState(null);
-  const [lastNameFilter, setLastNameFilter] = useState(null);
-  const [lastNameText, setLastNameText] = useState(null);
-  const [emailFilter, setEmailFilter] = useState(null);
-  const [emailText, setEmailText] = useState(null);
   const [hideLastNameAndEmailFilter, setHideLastNameAndEmailFilter] = useState(true);
   const [actionMenuParticipant, setActionMenuParticipant] = useState(null);
   const [anchorElement, setAnchorElement] = useState(false);
   const [activeModalForm, setActiveModalForm] = useState(null);
   const [tabValue, setTabValue] = useState(null);
-
   const [locations, setLocations] = useState([]);
+
+  const textReducer = (state,action)=>{
```

Comment:
I typically declare reducers outside of the component so it isn't recreated on re-render, small improvement what do you think?

Reply:
Seems like an improvement to me.
Thanks!